.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+before_script:
+  - ulimit -n 4096
+  - rm -rf ~/ci-cog
+  - mkdir -p ~/ci-cog
+  - cd ~/ci-cog
+  - git clone $CI_REPOSITORY_URL
+  - cd cog
+  - git checkout $CI_COMMIT_REF_NAME
+
+cog-meta-perf-browser-warrior-wandboard-mesa-wpe-weston:
+  tags:
+    - docker
+  script:
+    - wget -O ~/toolchain.sh https://wk-contrib.igalia.com/yocto/meta-perf-browser/browsers/nightly/sdk/wandboard-mesa/browsers-glibc-x86_64-core-image-weston-wpe-armv7at2hf-neon-wandboard-mesa-toolchain-1.0.sh
```

Comment:
As far as I can see those commands should run just fine on other GNU/Linux systems and nothing special is required. Why not putting those to run on Travis-CI, for example, or some other CI system which can run on GitHub PRs?

Reply:
It just took me a few minutes to write down a basic `.travis.yml` using this toolchain, see PR #119 (and note how the CI ran on that PR already).